docs/example-bpf.md

```diff
@@ -34,13 +39,22 @@ This section describes running bpfd and the example bpf programs on a local host
 When running bpfd, it can be run as a process or run as a systemd service.
 Examples run the same, independent of how bpfd is deployed, other than requiring `sudo` or not.
 
+> **_NOTE:_** When running bpfd as a systemd service, bpfd can only access files owned by the `bpfd`
```

Comment:
Wait is only a problem when running with Systemd? I feel like we'll have this issue when just running bpfd locally on a system without `sudo`

Reply:
I guess I didn't know we could do that. I also run as a systemd service (or test with `sudo ./target/debug/bpfd`). Don't you need sudo in the latter for CAP_BPF?

Reply:
Yes, just tried it, you need sudo:
```
$ ./target/debug/bpfd 
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: EPERM', bpfd/src/main.rs:63:79
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
Ahh nw I've never actually run bpfd using systemd :( 

If that's not something we support anyways you can ignore my comment